perfkitbenchmarker/linux_benchmarks/netperf_benchmark.py

```diff
@@ -377,40 +376,28 @@ def RunNetperf(vm, benchmark_name, server_ip, num_streams):
               'sending_thread_count': num_streams,
               'max_iter': FLAGS.netperf_max_iter or 1}
 
-  netperf_cmd = ('{netperf_path} -p {{command_port}} -j {verbosity} '
-                 '-t {benchmark_name} -H {server_ip} -l {length} {confidence}'
-                 ' -- '
-                 '-P ,{{data_port}} '
-                 '-o {output_selector}').format(
-                     netperf_path=netperf.NETPERF_PATH,
-                     benchmark_name=benchmark_name,
-                     server_ip=server_ip,
-                     length=FLAGS.netperf_test_length,
-                     output_selector=OUTPUT_SELECTOR,
-                     confidence=confidence,
-                     verbosity=verbosity)
+  netperf_cmd = (f'{netperf.NETPERF_PATH} -p {{command_port}} -j {verbosity} '
+                 f'-t {benchmark_name} -H {server_ip} -l {FLAGS.netperf_test_length} {confidence}'
```

Comment:
Hard to tell in the split screen view but this might have to be split into multiple lines.  Also I would find it easier to read
  f'-l {FLAGS.netperf_test_length} {confidence}'